Apps/Sandcastle/gallery/Particle System Weather.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
```

Comment:
Make sure you're thumbnail is sized correctly.

Reply:
Sandcastle provides a button to scale the output to the correct thumbnail size.  You just need a paint program that can auto-crop to that after a screen capture.

Reply:
resized.